src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -756,6 +756,7 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    matchFairsLoader: gravityLoader("match/fairs", {}, { headers: true }),
```

Comment:
Hits [this Gravity endpoint](https://github.com/artsy/gravity/blob/main/app/api/v1/match_endpoint.rb#L72-L84)